scripts/minikube.sh

```diff
@@ -160,7 +169,7 @@ CSI_RESIZER_VERSION=${CSI_RESIZER_VERSION:-"v1.2.0"}
 CSI_NODE_DRIVER_REGISTRAR_VERSION=${CSI_NODE_DRIVER_REGISTRAR_VERSION:-"v2.2.0"}
 
 #feature-gates for kube
-K8S_FEATURE_GATES=${K8S_FEATURE_GATES:-"ExpandCSIVolumes=true"}
+K8S_FEATURE_GATES=${K8S_FEATURE_GATES:-""}
```

Comment:
Now you are dropping `ExpandCSIVolumes` without comment. It is better to do that in a stand-alone commit. As a cleanup, it can be included in this PR, no need to create a separate one.

Reply:
sure, put it as a seperate commit here in this PR itself.